tests/tests.json

```diff
@@ -255,36 +255,44 @@
 	"params": {"70": 0.0},
 	"parallelization": ["mpi"]
     },
-    "drift-dirichlet-zaxis": {
+    "driftbcs-dirichlet-zaxis": {
 	"app": "drift",
 	"dir": "driftbcs",
 	"usr": "d_inhomo",
 	"rea": "d_inhomo",
 	"params": {},
 	"parallelization": ["mpi"]
     },
-    "drift-neumann-zaxis": {
+    "driftbcs-neumann-zaxis": {
 	"app": "drift",
 	"dir": "driftbcs",
 	"usr": "n_inhomo",
 	"rea": "n_inhomo",
 	"params": {},
 	"parallelization": ["mpi"]
     },
-    "drift-robin-zaxis": {
+    "driftbcs-robin-zaxis": {
 	"app": "drift",
 	"dir": "driftbcs",
 	"usr": "r_inhomo",
 	"rea": "r_inhomo",
 	"params": {},
 	"parallelization": ["mpi"]
     },
-    "drift-dirichlet-bdf1r": {
+    "driftbcs-dirichlet-bdf1r": {
 	"app": "drift",
 	"dir": "driftbcs",
 	"usr": "d_inhomo",
 	"rea": "d_inhomo",
 	"params": {"17": -11},
 	"parallelization": ["mpi"]
+    },
+    "newton-dirichlet": {
```

Comment:
Note that this test actually didn't get run on Travis because of the changes in https://github.com/NekCEM/NekCEM/commit/56cff361066b9af363e00d0ef2b8a402c9f2a5a4; tests need to be prefixed with their solver name now that that Travis can split them up appropriately. (See https://travis-ci.org/NekCEM/NekCEM/jobs/255896382.) Not a big deal though; we'll fix it in the morning.